src/chromeDebugAdapter.ts

```diff
@@ -347,6 +332,37 @@ export class ChromeDebugAdapter extends CoreDebugAdapter {
         this._chromeProc = null;
     }
 
+    private async killChromeOnWindows(chromePID: number): Promise<void> {
+        let taskkillCmd = `taskkill /PID ${chromePID}`;
+        logger.log(`Killing Chrome process by pid: ${taskkillCmd}`);
+        try {
+            execSync(taskkillCmd);
+        } catch (e) {
+            // The command will fail if process was not found. This can be safely ignored.
+        }
+
+        for (let i = 0 ; i < 10; i++) {
+            // Check to see if the process is still running
+            let tasklistOutput = execSync(`tasklist /FI "PID eq ${chromePID}"`).toString();
+            if (!tasklistOutput.includes(chromePID.toString())) {
```

Comment:
Is it possible that the chromePID is a substring of another process PID?

Reply:
When tasklist finds the process it will return output that looks like
`Image Name PID Session Name Session# Mem Usage`
`========== === ============ ======== === =====`
`chrome.exe 16116 Console 1 273,300 K`

When tasklist doesn't find the process it outputs
`INFO: No tasks are running which match the specified criteria.`

In both cases the exit code is 0

Because the tasklist is run with a PID filter it will only return one or zero processes. Any process that with a PID that is a superstring will be filtered out.

I implemented with a search of PID in the output because it seemed like the most robust way to differentiate between success and failure outputs. Searching for the presence of the PID is independent of the format.

Let me know if you have any better suggestions. I can add comments as documentation for what is going on.